openapi2kong/service.go

```diff
@@ -230,7 +230,11 @@ func CreateKongService(
 	if service["port"] == nil {
 		if targets[0].Port() != "" {
 			// port is provided, so parse it
-			service["port"], _ = strconv.ParseInt(targets[0].Port(), 10, 16)
+			parsedPort, err := strconv.ParseInt(targets[0].Port(), 10, 64)
```

Comment:
I think using `ParseUint` with bitsize `16` should do it right? That would also ensure any negative numbers are caught by the error handler.